pybossa/importers/csv.py

```diff
@@ -185,8 +275,27 @@ def _check_required_headers(self):
                           'required header(s): {0}'.format(','.join(missing_headers)))
             raise BulkImportException(msg)
 
-    def _get_csv_data_from_request(self, r):
+    # must implement
```

Comment:
This is used here https://github.com/bloomberg/pybossa/pull/334/files#diff-baa69587524fb0513758375c31748d5bR188 do you need it?

Reply:
Yes, it is needed. It's an abstract method, so the comment was just to say that the derived class must implement it.

Reply:
you can define the method and in it `raise NotImplementedError()`

Reply:
Did it.